packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Role Assignment
+---
+<ApiLifecycle access="beta" />
+
+# Role Assignment
+Role assignment to principals makes them administrators of your org. Principals can be users or groups of users.
+
+## Standard Role Assignment
+The following role types are provided and supported out of the box:
+| Role type                               | Label                               | Optional targets                      |
+| :-------------------------------------- | :---------------------------------- | :------------------------------------ |
+| `API_ACCESS_MANAGEMENT_ADMIN`           | API Access Management Administrator |                                       |
+| `APP_ADMIN`                             | Application Administrator           | Apps                                  |
+| `GROUP_MEMBERSHIP_ADMIN`                | Group Membership Administrator      | [Groups](/docs/reference/api/groups/) |
+| `HELP_DESK_ADMIN`                       | Help Desk Administrator             | [Groups](/docs/reference/api/groups/) |
+| `MOBILE_ADMIN`                          | Mobile Administrator                |                                       |
+| `ORG_ADMIN`                             | Organizational Administrator        |                                       |
+| `READ_ONLY_ADMIN`                       | Read-Only Administrator             |                                       |
+| `REPORT_ADMIN`                          | Report Administrator                |                                       |
+| `SUPER_ADMIN`                           | Super Administrator                 |                                       |
+| `USER_ADMIN`                            | Group Administrator                 | [Groups](/docs/reference/api/groups/) |
+
+Standard role assignment is done in two steps:
+1. Assign a role to a user or group. At this point the admin has the supported privileges of the role over all resources across organization.
```

Comment:
```suggestion
1. Assign a Role to a User or Froup. At this point the admin has the supported privileges of the role over all resources across organization.
```

@alifatolahi-okta we capitalize the names of objects within Okta so as to distinguish them from the thing itself. So a User object models a user, a Role object contains information about a role, etc.

Could you ensure that all instances of object names in these docs are capitalized?

Reply:
done